tests/resource_tests/schema_tests/test_resource_schemas.py

```diff
@@ -0,0 +1,258 @@
+        self._verify_resource_schema('GenomicJobRunSchema',
+                                     rschemas.GenomicJobRunSchema(),
+                                     bq_genomics.BQGenomicJobRunSchema(), bq_prefix="orig_")
+
+    def test_genomic_file_processed_resource_schema(self):
+        self._verify_resource_schema('GenomicFileProcessedSchema',
+                                     rschemas.GenomicFileProcessedSchema(),
+                                     bq_genomics.BQGenomicFileProcessedSchema(), bq_prefix="orig_")
+
+    # TODO:  Confirm relationship of the id and orig_id fields (and/or similar created, modified) in genomic schemas
+    def test_genomic_manifest_file_resource_schema(self):
+        self._verify_resource_schema('GenomicManifestFileSchema',
+                                     rschemas.GenomicManifestFileSchema(),
+                                     bq_genomics.BQGenomicManifestFileSchema())
+
+    def test_genomic_manifest_feedback_resource_schema(self):
+        self._verify_resource_schema('GenomicManifestFeedbackSchema',
+                                     rschemas.GenomicManifestFeedbackSchema(),
+                                     bq_genomics.BQGenomicManifestFeedbackSchema())
+
+    def test_genomic_gc_validation_metrics_resource_schema(self):
+        self._verify_resource_schema('GenomicGCValidationMetricsSchema',
+                                     rschemas.GenomicGCValidationMetricsSchema(),
+                                     bq_genomics.BQGenomicGCValidationMetricsSchema())
+
+    # Researcher workbench related schemas
+
+    def test_rwb_researcher_resource_schema(self):
+        self._verify_resource_schema('WorkbenchResearcherSchema',
+                                     rschemas.workbench_researcher.WorkbenchResearcherSchema(),
+                                     bq_workbench_researcher.BQRWBResearcherSchema())
+
+    def test_rwb_institutional_affiliations_resource_schema(self):
+        self._verify_resource_schema('WorkbenchInstitutionalAffiliationsSchema',
+                                     rschemas.workbench_researcher.WorkbenchInstitutionalAffiliationsSchema(),
+                                     bq_workbench_researcher.BQRWBInstitutionalAffiliationsSchema())
+
+    def test_workbench_workspace_resource_schema(self):
+        self._verify_resource_schema('WorkbenchWorkspaceSchema',
+                                     rschemas.workbench_workspace.WorkbenchWorkspaceSchema(),
+                                     bq_workbench_workspace.BQRWBWorkspaceSchema())
+
+    def test_workbench_workspace_user_resource_schema(self):
+        self._verify_resource_schema('WorkbenchWorkspaceUserSchema',
+                                     rschemas.workbench_workspace.WorkbenchWorkspaceUsersSchema(),
+                                     bq_workbench_workspace.BQRWBWorkspaceUsersSchema())
+
+    @unittest.skip('Confirm if WorkspaceAgeSchema needs a SchemaID defined')
```

Comment:
It's not clear to me if we need new/separate resource_types for these (and separate resource APIs);  will address in a future PR if we do.

Reply:
Sounds good.